core/src/main/scala/cats/syntax/all.scala

```diff
@@ -31,6 +31,7 @@ trait AllSyntax
     with EqSyntax
     with FlatMapSyntax
     with FoldableSyntax
+    with Function1Syntax
```

Comment:
due to binary compatibility restrictions, you can't add it to this trait either. However, you can add it to `AllSyntaxBinCompat3 ` which is a new trait that hasn't been released yet.

Reply:
Thanks. 👍